eahub/templates/emails/message_profile.html

```diff
@@ -0,0 +1,12 @@
+<p>Dear {{ recipient }},</p>
+<p>{{ sender_name }} has sent you a message:</p>
+<p>{{ message }}</p>
+<p>Thanks,</p>
+<p>EA Hub</p>
+<p>
+  <a href="feedback.eahub.org">Feedback</a>
+  <span> | </span>
+  <a href="eahub.org/profile/">Messaging Preferences</a>
```

Comment:
we need a `{% url %}` tag here too

Reply:
also hardcoding eahub.org isn't safe for various reasons, it has to be a variable

Reply:
You're ofc right. Replaced all of the hard coded values with variables